src/views/Send/Send.vue

```diff
@@ -642,19 +641,24 @@ export default {
       const { amount, address, selectedFee, currentStep, maxOptionActive, customFee } = sendParams
       this.amount = amount
       this.address = address
+      if (maxOptionActive) {
+        this.maxOptionActive = maxOptionActive === 'true' ? true : false
```

Comment:
@bradleySuira why aren't we doing `this.maxOptionActive = maxOptionActive` here though?

Reply:
it's because the library that we are using `qs` when parse the data back convert all values to string, it's really handy but have that issue